dev/shell.nix

```diff
@@ -6,6 +6,7 @@
         jq
         python3.pkgs.deploykit
         python3.pkgs.invoke
+        python3.pkgs.mkdocs-material
```

Comment:
Removed this in https://github.com/nix-community/infra/commit/486311ad294ec325796f99e78bccbb7b991e261b to reduce bloat, can build docs locally with `inv mkdocs`.

Reply:
Removed